### ISSUE TITLE
Add test suite to very build?

### ISSUE THREAD

**Issue by petterreinholdtsen** [dag-erling/conv-tools#1]
It would be nice if the package verified that it was working when doing 'make check'.  Here is a simple script to test the dirconv program:

```
#!/bin/sh

set -e
cleanup() {
    if [ "$SCRATCHDIR" ] ; then
    rm -rf "$SCRATCHDIR"
    SCRATCHDIR=""
    fi
}
SCRATCHDIR=$(mktemp -d)
trap cleanup EXIT

oname=$(printf "\330.txt")
nname=$(printf "Ø.txt")

touch "$SCRATCHDIR/$oname"
ls -lR $SCRATCHDIR

bin/dirconv/dirconv -r $SCRATCHDIR

ls -lR $SCRATCHDIR

if [ ! -e "$SCRATCHDIR/$nname" ] ; then
    echo "error: failed to find new file name."
    exit 1
else
    echo "success: file name converted correctly."
fi
```

Perhaps something like this could be added to the package?

**Comment by dag-erling** [comment]
Both `dirconv` and `mixconv` already have self-test modes, they just need to be hooked up to the build.

**Comment by dag-erling** [comment]
Fixed in #cb5a4c6 and #2b781ac
